lib/zone/zone.ts

```diff
@@ -3,18 +3,40 @@
 
 import * as transport from "../transport";
 
+/// <summary> Represent option when the zone is recycled by GC. </summary>
+export enum ZoneRecycleMode {
+
+    /// <summary>
+    /// When a Zone is no longer referenced and all tasks on its workers are completed,
+    /// it will be recycled automatically.
+    /// </summary>
+    AUTO = "auto",
+
+    /// <summary>
+    /// The zone will not exit automatically.
+    /// User should call zone.recycle() manually.
+    /// </summary>
+    MANUAL = "manual",
+}
+
 /// <summary> Describes the available settings for customizing a zone. </summary>
 export interface ZoneSettings {
 
     /// <summary> The number of workers that will serve zone requests. </summary>
     workers?: number;
+
+    /// <summary> The zone recycling option. </summary>
+    recycle?: ZoneRecycleMode;
 }
 
 /// <summary> Default ZoneSettings </summary>
 export let DEFAULT_SETTINGS: ZoneSettings = {
 
-    /// <summary> Set default workers to 2. </summary>
-    workers: 2
+    /// <summary> Set default workers to 1. </summary>
+    workers: 1,
```

Comment:
>1 [](start = 13, length = 1)

Is there any test cases using this default value? If yes, it will effect the tests behavior, which we should pay some attention.